src/commands/commands.service.ts

```diff
@@ -33,13 +33,14 @@ export class CommandsService implements OnModuleInit {
 			const commandsByGuildMap = new Map<string, Array<CommandDiscovery>>([[undefined, []]]);
 
 			for (const command of commands) {
+				const commandGuilds = command.getGuilds();
 				const defaultGuild = Array.isArray(this.options.development)
 					? this.options.development
 					: [undefined];
 
-				for (const guild of defaultGuild) {
-					const visitedCommands = commandsByGuildMap.get(guild) ?? [];
-					commandsByGuildMap.set(guild, visitedCommands.concat(command));
+				for (const guildId of commandGuilds.length ? commandGuilds : defaultGuild) {
```

Comment:
So, what do u think about using guilds specified by @Guilds only in production?
Will be two cases:
- If specify `development` (DEV) - all commands register to dev guild, ignore @Guilds
- If don't specify `development`(PROD) - All commands register global, exclude guilds commands by @Guilds

Reply:
But what happens when I am in dev and do not specify the `development` configuration in `NecordModule`? Maybe set the `development` configuration required?

Reply:
If you in dev, you need to set `development`, but maybe it's a good idea for the required property

Reply:
Good, because I do not have the `development` set here, hehe.

So, what do you think that I have to change in the code to apply the `@Guilds` only in production?

Reply:
Maybe something like:

```ts
const guilds = Array.isArray(this.options.development)
	? this.options.development
	: command.getGuilds() ?? [undefined];
```

Reply:
Done ✅